openhtmltopdf-core/src/main/java/com/openhtmltopdf/simple/extend/XhtmlCssOnlyNamespaceHandler.java

```diff
@@ -395,10 +395,18 @@ public StylesheetInfo getDefaultStylesheet(StylesheetFactory factory) {
         }
     }
 
+    /**
+     * @return path to the default css-File in the classpath
+     */
+    protected String getPathToDefaultStylesheet() {
+        return PATH_TO_DEFAULT_STYLESHEET;
+    }
+
     private InputStream getDefaultStylesheetStream() {
-        InputStream stream = this.getClass().getResourceAsStream(PATH_TO_DEFAULT_STYLESHEET);
+        final String pathToStylesheet = getPathToDefaultStylesheet();
+        final InputStream stream = this.getClass().getResourceAsStream(pathToStylesheet);
```

Comment:
```suggestion
        String pathToStylesheet = getPathToDefaultStylesheet();
        InputStream stream = this.getClass().getResourceAsStream(pathToStylesheet);
```
Please don't use `final` declarations on local variables as the rest of the code does not follow this style.